src/app/modules/submission/multirespondent-submission/multirespondent-submission.middleware.ts

```diff
@@ -796,6 +799,18 @@ export const encryptSubmission = async (
       req.body.version,
     )
 
+  // Modify response data for webhook responses and encrypt separately
+  const strippedResponsesWebhook = prepareWebhookResponseContent(
```

Comment:
praise: nice use of abstraction to keep the middleware simple to understand 

- wondering if we could also do this for encrypt mode - by perhaps reusing the `prepareWebhookResponseContent` function?